lib/seahorn/BvOpSem2.cc

```diff
@@ -149,6 +150,30 @@ static llvm::cl::opt<bool>
                     llvm::cl::desc("Use crab to infer rng invariants"),
                     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> UseCrabLowerIsDeref(
+    "horn-bv2-crab-lower-is-deref",
+    llvm::cl::desc("Use crab to lower sea.is_dereferenceable"),
+    llvm::cl::init(false));
+
+static llvm::cl::opt<clam::CrabDomain::Type, false, clam::CrabDomainParser>
```

Comment:
`PathBmc.cc` already defines a command-line option to choose the abstraction domain. 
Replace this definition of `CrabDom` with this code:

```
namespace seahorn {
  extern clam::CrabDomain::Type CrabDom;
}
```